src/time_zone_info.cc

```diff
@@ -551,7 +551,7 @@ std::unique_ptr<ZoneInfoSource> FuchsiaZoneInfoSource::Open(
   const auto prefixes = name_absolute ? kEmptyPrefix : kTzdataPrefixes;
 
   // Fuchsia builds place zoneinfo files at "<prefix><format><name>".
-  for (const std::string prefix : prefixes) {
+  for (const std::string& prefix : prefixes) {
```

Comment:
This change is actually incorrect. prefixes is an array of `const char[]`, so this binds a temporary.